From bc2ba0b30896a52e632cdf13f95b9900816e9d6a Mon Sep 17 00:00:00 2001
From: Chang Rebecca Swee Fun <rebecca.swee.fun.chang@intel.com>
Date: Thu, 9 Jun 2016 02:23:01 -0700
Subject: [PATCH 1/7] ovmf: update path to native BaseTools

BaseTools is a set of utilities to build EDK-based firmware. These utilities
are used during the build process. Thus, they need to be built natively.
When cross-compiling, we need to provide a path to the location of these
tools. We now directly point it to $STAGING_BINDIR_NATIVE which will be
expanded before building.

Upstream-Status: Inappropriate [OE specific]

Signed-off-by: Chang Rebecca Swee Fun <rebecca.swee.fun.chang@intel.com>

---
 OvmfPkg/build.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/OvmfPkg/build.sh b/OvmfPkg/build.sh
index 6821552..55922c5 100755
--- a/OvmfPkg/build.sh
+++ b/OvmfPkg/build.sh
@@ -30,7 +30,7 @@ then
 # this assumes svn pulls have the same root dir
 #  export EDK_TOOLS_PATH=`pwd`/../BaseTools
 # This version is for the tools source in edk2
-  export EDK_TOOLS_PATH=`pwd`/BaseTools
+  export EDK_TOOLS_PATH=$STAGING_BINDIR_NATIVE/edk2_basetools/BaseTools
   echo $EDK_TOOLS_PATH
   source edksetup.sh BaseTools
 else
-- 
2.7.4

